nusight2/src/client/components/localisation/view.tsx

```diff
@@ -201,3 +202,19 @@ const FieldLineDots = ({ model }: { model: LocalisationModel }) => (
     )}
   </>
 );
+
+const Ball = ({ model }: { model: LocalisationModel }) => (
+  <>
+    {model.robots.map(
+      (robot) =>
+        robot.visible && (
+          <object3D key={robot.id}>
+            <mesh position={robot.ball.toArray()}>
+              <sphereBufferGeometry args={[robot.ball.z, 20, 20]} />
+              <meshStandardMaterial color="#1E90FF" />
+            </mesh>
+          </object3D>
```

Comment:
No need for an `Object3D` container here since we only have one mesh:

```suggestion
          <mesh position={robot.ball.toArray()} key={robot.id}>
            <sphereBufferGeometry args={[robot.ball.z, 20, 20]} />
            <meshStandardMaterial color="#1E90FF" />
          </mesh>
```